_source/user-guide/alerts/configure-alerts.md

```diff
@@ -85,6 +85,28 @@ In the _Trigger if..._ section, click **+ Add a threshold** to set up to 5 thres
 
 ![Alert trigger thresholds](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/alerts--trigger-settings_aug2021.png)
 
+##### Set alert schedule
+
+You can use the scheduling mechanism to manage the trigger condition frequency.
+
+Scheduling defines the frequency and the time frame for the alerts. To define a schedule, tick the On Schedule toggle and use [cron expression](https://www.freeformatter.com/cron-expression-generator-quartz.html) to specify when to trigger the alert.
+
+<!-- info-box-start:info -->
+You cannot set the cron expression to a value under a minute, and you must use round minutes.
+{:.info-box.note}
+<!-- info-box-end -->
+
+For example, you can apply the following schedule to your alerts:
+
+| Cron expression                         | Alert trigger schedule |
+|-----------------------------------------|------|
+| 0 0/10 * ? * * *                        | Every 10 minutes |
+| 0 0 0/1 ? * * *                         | Every round hour |
+| 0 * 8-17 ? * MON,TUE,WED,THU,FRI *  | Every minute between 8 am to 5 pm, Monday through Friday |
+| 0 5 0 ? * * *                           | Every day at exactly 12:05 am |
+
+By default, trigger conditions run approximately every minute. If there's a lag, the alert will not trigger until all data reaches Elastic. Also, once an alert is triggered, it won't run again for the remainder of the alert trigger condition.
```

Comment:
"the alert will not trigger until all data reaches Elastic."

> The alert is not triggered until all the data reaches Logz.io 

The last sentence is unclear.  Is the intention that the trigger condition is not checked again for the duration of some period of time? Or that